app/src/composables/use-relation-multiple.ts

```diff
@@ -196,8 +218,19 @@ export function useRelationMultiple(
 		return { create, update, remove, select, clear };
 
 		function create(...items: Record<string, any>[]) {
+			const sortField = relation.value?.sortField;
+
 			for (const item of items) {
-				target.value.create.push(cleanItem(item));
+				target.value.create.push(
+					cleanItem(
+						sortField && (item[sortField] === undefined || item[sortField] === null)
```

Comment:
`isNil` from lodash does the same as `item[sortField] === undefined || item[sortField] === null`  and is a bit cleaner.